core/src/banking_stage.rs

```diff
@@ -172,7 +172,9 @@ impl BankingStage {
             }
             rebuffered_packets += new_unprocessed_indexes.len();
```

Comment:
Should you also not increment `rebuffered_packets`?

Reply:
It will increment with 0, but I guess as long as we are already doing the empty check might as well save the add I suppose.